sensor_msgs/CompressedImage image
---
sensor_msgs/Image segmentation